webpack/assets/javascripts/react_app/redux/API/APIRequest.js

```diff
@@ -22,6 +22,10 @@ export const apiRequest = async (
 ) => {
   const { REQUEST, SUCCESS, FAILURE } = actionTypeGenerator(key, actionTypes);
   const modifiedPayload = { ...payload, url };
+  const stopIntervalCallback = selectDoesIntervalExist(getState(), key)
+    ? () => dispatch(stopInterval(key))
+    : noop;
```

Comment:
how about making a function that prints to the console a warning if a user consumes it?

Reply:
great idea, added :)